recipes/eups/activate.sh

```diff
@@ -0,0 +1,5 @@
+# Bootstrap EUPS
+EUPS_DIR="${CONDA_PREFIX}/eups"
+source "${EUPS_DIR}/bin/setups.sh"
```

Comment:
This is going to stomp on any existing eups installs that are active. So for example, if a user has a global eups distrribution installed outside of conda, this command will mess that up. You need to backup any env vars that setups.sh is going to override.

Reply:
You probably need to backup any other envvars eups might touch too. This could include LD_LIBRARY_PATH, DYLD_LIBRARY_PATH on osx, etc.

Reply:
One more thing here. The activate and deactivate scripts are called in lexicographical order. This means this script will be called after clang but before any gnu compilers. This is something to think about. You might want to inspect the compiler activation packages to figure out what you might be stepping on. Generally stepping on flags from them is a no-no as they have to be intact for the system to work. 

As an aside, I got lucky when I did this because `s` comes after `c` and `g`! ;p

Reply:
Unfortunately, eups is general enough that it can modify any arbitrary environment variable.  Should we attempt to preserve and restore the entire _status quo ante_ environment (e.g. in a special file)?

Reply:
Ah I was confused between eups and eups distrib. Let’s just preserve the env vars explicitly uses by eups itself. We can punt the preservation of env vars related to specific packages to those packages.